libs/cart/src/facades/cart/cart.facade.ts

```diff
@@ -179,6 +185,10 @@ export class DaffCartFacade<
     this.shippingInformation$ = this.store.pipe(select(selectCartShippingInformation));
     this.availableShippingMethods$ = this.store.pipe(select(selectCartAvailableShippingMethods));
     this.availablePaymentMethods$ = this.store.pipe(select(selectCartAvailablePaymentMethods));
+    this.paymentId$ = this.payment$.pipe(
```

Comment:
I think this should return null if `!!(payment && payment.method)`. Is there a reason it doesn't return anything instead?

Reply:
Nah no particular reason. changed